valheim_plus.cfg

```diff
@@ -248,6 +248,9 @@ disableCameraShake=false
 ; The base unarmed damage multiplied by your skill level
 baseUnarmedDamage=10
 
+; When changed to true, makes you unable to place crops too close to eachother
+cropNotifier=false
```

Comment:
I would put a bit more clear wording on the description. Maybe: "When changed to true, you will not be permitted to place a crop that is considered to close to another crop"